tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php

```diff
@@ -342,24 +500,30 @@ private function mapDataToEditableCategory(
         if (isset($testCaseData['Description'])) {
             $description = array($this->defaultLanguageId => $testCaseData['Description']);
         }
-
         $metaTitle = array($this->defaultLanguageId => self::EMPTY_VALUE);
         if (isset($testCaseData['Meta title'])) {
             $metaTitle = array($this->defaultLanguageId => $testCaseData['Meta title']);
         }
-
         $metaDescription = array($this->defaultLanguageId => self::EMPTY_VALUE);
         if (isset($testCaseData['Meta description'])) {
             $metaDescription = array($this->defaultLanguageId => $testCaseData['Meta description']);
         }
-
         $linkRewrite = array($this->defaultLanguageId => self::EMPTY_VALUE);
         if (isset($testCaseData['Friendly URL'])) {
             $linkRewrite = array($this->defaultLanguageId => $testCaseData['Friendly URL']);
         }
-
-        if ($parentCategoryId === null) {
-            $parentCategoryId = CategoryTreeIterator::ROOT_CATEGORY_ID;
+        if (isset($testCaseData['Category cover image'])) {
+            $this->pretendImageUploading($testCaseData, $categoryId);
+        }
+        if (isset($testCaseData['Menu thumbnails'])) {
+            $categoryCoverImage = $testCaseData['Menu thumbnails'];
+            // could not use handler because it uses move_uploaded_file in Uploader.php which allows only POST upload
+            /** @var Kernel $kernel */
+            $kernel = $this->getContainer()->get('kernel');
+            copy(
+                $kernel->getRootDir() . '/../img/' . $categoryCoverImage,
+                _PS_CAT_IMG_DIR_ . $categoryId . '-' . MenuThumbnailId::ALLOWED_ID_VALUES[0] . '_thumb.jpg'
+            );
```

Comment:
same here as in comment https://github.com/PrestaShop/PrestaShop/pull/17018/files#r362855042

Reply:
Thanks Tomas, for now commented out scenarios using this method. While will figure out better approach.